codespell_lib/data/dictionary.txt

```diff
@@ -21118,13 +21161,14 @@ similiarity->similarity
 similiarly->similarly
 similiarty->similarity
 similiary->similarity
+simily->simile, smiley, similarly, simply,
```

Comment:
I think `similarly, simply` should swap order

Reply:
bce81df